go.mod

```diff
@@ -164,15 +164,15 @@ require (
 	go.opentelemetry.io/otel/sdk v1.16.0
 	go.opentelemetry.io/otel/trace v1.16.0
 	go.opentelemetry.io/proto/otlp v1.0.0
-	golang.org/x/crypto v0.12.0
+	golang.org/x/crypto v0.14.0
```

Comment:
You'll want to pull in #33169 along with the new crypto.

Reply:
Thank you @codingllama, just pulled that change in.